juniper_official/Protocols/Isis/isis-statistics.rule

```diff
@@ -72,6 +75,7 @@
                 sensor isis-sensor {
                     where "/network-instances/network-instance/protocols/protocol/isis/interfaces/interface/@interface-id =~ /{{interface-name}}/";
                     path "/network-instances/network-instance/protocols/protocol/isis/interfaces/interface/@interface-id";
+                    zero-suppression;
```

Comment:
Why are we zero-suppressing something of type string? Can you please double check this?

Reply:
Corrected.